### ISSUE TITLE
how did you manage to get Fan read-out with SMCSuperIO?

### ISSUE THREAD

**Issue by kasti0** [gajjartejas/Lenovo-Ideapad-320-15ISK-14ISK-Laptop-Hackintosh#36]
i have a very similar laptop the lenovo ideapad 330s 15ikb. I can´t get fan read-out by just adding the SMCSuperIO.kext. did you do any other patches to achieve this?
i´m using OpenCore 0.6.4, never had fan read-out on any macOS version

**Comment by gajjartejas** [comment]
As far as I know, SMCSuperIO only for desktop, you should try FakeSMC instead. Right now my laptop Ideapad 320 15ISK does not support fan control.

**Comment by kasti0** [comment]
So you just don’t have fan control or not any fan read-out at all?
Seems like there’s some SSDT writing necessary to get full control over the fans

**Comment by gajjartejas** [comment]
No, I don't have any fan control or fan reading. I think my laptop does not support it. Do you have fan control?

**Comment by kasti0** [comment]
Unfortunately not. It seems like there is a new kext called YogaSMC that tries to add as much functionality as possible to the Lenovo lineup (Yoga, Thinkpad, ideapad)

**Comment by gajjartejas** [comment]
Okay thanks for information, I'll take a look into it.

**Comment by gajjartejas** [comment]
I have installed YogaSMC in the first screenshot function key mode working.
![Screenshot 2020-11-23 at 5 47 08 PM](https://user-images.githubusercontent.com/14052969/99961232-08353500-2db4-11eb-9dde-d31059364c2c.png)

For the second screenshot battery conservation mode and temperature working.
![Screenshot 2020-11-23 at 5 47 02 PM](https://user-images.githubusercontent.com/14052969/99961217-04a1ae00-2db4-11eb-91d8-cab55e64b583.png)
